README.md

```diff
@@ -1,36 +1,138 @@
 
-### Downloading OpenGL
+## installation
 
-Windows and Linux users may need to install the [Mesa OpenGL](https://www.mesa3d.org/) library.
+napari can be installed on most Mac OS X and Linux systems with Python 3.6 or 3.7 by calling 
 
-`vispy` provides an easily-downloadable (but out-of-date) Windows version for [64-](https://github.com/vispy/demo-data/raw/master/mesa/opengl32_mingw_64.dll) and [32-bit](https://github.com/vispy/demo-data/raw/master/mesa/opengl32_mingw_64.dll) operating systems.
+```sh
+$ pip install napari
+```
 
-Linux users can complete the installation process from the command line:
+We're working on adding Windows support.
 
-```sh
-$ sudo apt update
-$ sudo apt install libgl1-mesa-glx
+## simple example
+
+From inside the IPython shell or a Jupyter notebook you can run the following to open up an interactive viewer
+
+```python
+%gui qt5
+from skimage import data
+from napari import ViewerApp
+viewer = ViewerApp(data.astronaut())
+```
+
+![image](resources/screenshot-add-image.png)
+
+To do the same thing inside a script call
+
+```python
+from skimage import data
+from napari import ViewerApp
+from napari.util import app_context
+
+with app_context():
+	viewer = ViewerApp(data.astronaut())
+```
+
+## more features
+
+Check out the scripts in the `examples` folder to see some of the functionality we're developing!
+
+For example, you can add multiple images in different layers and adjust them
+
+```python
+from skimage import data
+from skimage.color import rgb2gray
+from napari import ViewerApp
+from napari.util import app_context
+
+with app_context():
+    viewer = ViewerApp(astronaut=rgb2gray(data.astronaut()),
+                       photographer=data.camera(),
+                       coins=data.coins(),
+                       moon=data.moon())
+
+    viewer.layers.remove('coins')
+    viewer.layers['astronaut', 'moon'] = viewer.layers['moon', 'astronaut']
+```
+
+![image](resources/screenshot-layers.png)
+
+You can add markers on top of an image
+
+```python
+from skimage import data
+from skimage.color import rgb2gray
+from napari import ViewerApp
+from napari.util import app_context
+
+with app_context():
+    viewer = ViewerApp()
+    viewer.add_image(rgb2gray(data.astronaut()))
+    points = np.array([[100, 100], [200, 200], [333, 111]])
+    size = np.array([10, 20, 20])
+    markers = viewer.add_markers(points, size=size)
+```
+
+![image](resources/screenshot-add-markers.png)
+
+napari support bidirectional communication between the viewer and the Python kernel, which is especially useful in Jupyter notebooks -- in the example above you can retrieve the locations of the markers, including any additional ones you have drawn, by calling
+
+```python
+markers.coords
+>> [[100 100]
+    [200 200]
+    [333 111]]
+```
+
+Finally, you can render and quickly browse slices of multi-dimensional arrays
+
+```python
+
+import numpy as np
+from skimage import data
```

Comment:
Sorry to harp on this point, but I think we should add skimage to the requirements if we are going to use it in a readme example.

Based on the discussion in #179 and in the slack, I think there is some concern about `skimage`'s `matplotlib` dependency. However, given the number of examples that rely on the skimage, I think most new users are going to have to install skimage to get off the ground anyway. At least if we include it in the `requirements.txt`, we can pin it to a version we know is compatible with `napari`. Thoughts?

Reply:
definitely good to resolve this one, but maybe not as part of this PR? my personal view is that we should pull out the example data we want / need and should make it easy to pull that in with `napari` rather than a hard dependence on `skimage` -- but again, seems like something to resolve separately? unless for now we want to say in the readme something like:

> You can additionally `pip install skimage` to get the same data for the examples